addons/mail/static/src/new/discuss/discuss.xml

```diff
@@ -38,6 +38,13 @@
                         <button t-if="!thread.inCall" class="btn" title="Start a Call" t-on-click="startCall">
                             <i class="fa fa-lg fa-phone text-700"/>
                         </button>
+                    </t>
+                    <t t-if="thread.type === 'channel'">
+                        <button class="btn" t-attf-title="{{state.activeMode === 'memberList' and 'Hide' or 'Show'}} Member List" t-on-click.stop="toggleMemberList">
```

Comment:
- Prefer ternary operator `:` and `?`. It's less error-prone than relying on precedence order of `and` and `or` (and also not resulting to something falsy/truthy by mistake, it guarantees the result is an explicit string)
- This string is not `l10n` compliant. You should have 2 different `l10n` string for `Show Member List` and `Hide Member List`.